jest-puppeteer.config.js

```diff
@@ -4,6 +4,7 @@ const config = {
   launch: {
     slowMo: 10,
     headless: true,
+    ignoreHTTPSErrors: true,
```

Comment:
@rupl I added this so I could run the tests on my local https://commondesign.test
Otherwise I get `net::ERR_CERT_AUTHORITY_INVALID at https://commondesign.test`

Reply:
Yup that sounds fine 👍